ra/ra.go

```diff
@@ -1789,6 +1792,44 @@ func (ra *RegistrationAuthorityImpl) countFailedValidation(ctx context.Context,
 		}
 		ra.log.Warningf("spending against the %s rate limit: %s", ratelimits.FailedAuthorizationsPerDomainPerAccount, err)
 	}
+
```

Comment:
We'll want to use a feature flag to gate off this ratelimit so that we can administratively disable lines 1796-1820. This is different than disabling the new ratelimit by not having a default ratelimit defined because in this case we'd still emit RA warning logs.